### DISCUSSION TITLE
how can i change the image assets public path alone?

### DISCUSSION THREAD

**Discussion by Taoja** [webpack/webpack#15820] [Q&A]
i using webpack4+
now, the image assets path in chunk is public path + assets name like this:
```js
/***/ "./src/assets/login/logo.png?012e5df7379fece1f0e59d488c0e381b":
/***/ (function(module, exports, __webpack_require__) {

module.exports = __webpack_require__.p + "assets/logo.png";

/***/ }),
```

i wanna chang it to :
``` js
/***/ "./src/assets/login/logo.png?012e5df7379fece1f0e59d488c0e381b":
/***/ (function(module, exports, __webpack_require__) {

module.exports = __webpack_require__.__custom_path__ + "assets/logo.png";

/***/ }),
```

**Comment by vankop** [comment]
you can use https://webpack.js.org/configuration/module/#rulegeneratorpublicpath , note this is static public path..

**Reply by Taoja** [reply]
> you can use https://webpack.js.org/configuration/module/#rulegeneratorpublicpath , note this is static public path..

thx for ur help, but i wanna make the actual public path depends on chunk.name, and i using webpack 4+
